_config.yml

```diff
@@ -19,6 +19,7 @@ url: "https://gpuinspector.dev" # the base hostname & protocol for your site, e.
 project-url: "https://github.com/google/agi"
 releases-url: "https://github.com/google/agi/releases"
 docurl: "https://developer.android.com/agi"
+validation-url: "https://developer.android.com/agi/supported-devices"
```

Comment:
I think "https://developer.android.com/agi/start#device-validation" would be a better link to forward people for validation related information?

Reply:
I wonder if we should move the whole Device Validation subsection into the Supported Devices section?

Reply:
I think validation warrants its own page to make it easy to skim and find (including validation process description, table, and troubleshooting info in one page) -- I'll leave it at https://developer.android.com/agi/start#device-validation for now (since there are links to and from each page) until we adjust the docs

Reply:
I'd prefer having Supported Devices in a separate page but 'Device Validation' subsection can be moved out of 'Quickstart' into its own page. That way we can add more explanation related to the validation error messages and any known issues.